doc/governance.rst

```diff
@@ -86,6 +86,17 @@ For this, they can operate the scikit-learn accounts on various social networks
 and produce materials. They also have the required rights to our blog
 repository and other relevant accounts and platforms.
 
+Documentation team
+~~~~~~~~~~~~~~~~~~
+
+Members of the documentation team engage with the documentation of the project
+among other things. They might also be involved in other aspects of the
+project, but their reviews on documentation contributions of others are
+considered authoritative, and can merge such contributions.
+
+To this end, they have the required permissions to merge pull requests on the
+main repo.
```

Comment:
```suggestion
They have permissions to merge pull requests upstream.
```

Reply:
I preferred the original formulation: "main repo" or "main scikt-learn repo" more specific.

"upstream" can mean different things in different context (e.g. fixing something upstream is generally related to fixing something in a project that scikit-learn has a dependency on).

Reply:
How about this?
```suggestion
They have permissions to merge pull requests in scikit-learn's repository. 
```